src/styles/utils/variables.scss

```
@@ -0,0 +1,9 @@
+$c-blue: #00acdc;
```

Do not abbreviate the names to unreadable ones and try not to use the names according to the content, but use them according to the meaning, as you named most of the variables.
```suggestion
$blue-color: #00acdc;
```